src/payment.js

```diff
@@ -160,11 +160,11 @@ export class Payment {
      * @todo Check working-ness for BFF + SPiD
      * @param {string} campaignId
      * @param {string} productId
-     * @param {string} voucherCode
+     * @param {string} [voucherCode]
```

Comment:
Ok, now I'm **almost** happy, I promise 😄

My only remaining question is; should we not either:

1. Make `campaignId` and `productId` optional in the jsdoc (by adding `[]` around the `@param` args), or
2. Verify that the user supplies values there in the implementation

?

Reply:
campaign id and product id are required so I add validation for those